docs/GettingStarted.md

```diff
@@ -109,9 +109,8 @@ Now press RETURN here to jump into ONIE''s manual installer mode.  You should se
 Then simply download the latest ONL installer for the appropriate
 architecture (powerpc or amd64) from the website and run it.
 
-        ONIE:/ # install_url http://opennetlinux.org/binaries/latest-amd64.installer
+        ONIE:/ # install_url http://opennetlinux.org/binaries/latest-($ARCH).installer
```

Comment:
this is still not quite right.  Can you make this  `install_url http://opennetlinux.org/binaries/latest-$ARCH.installer` without the parentheses?  That way people can just cut and paste it.